src/averageAges.js

```diff
@@ -37,7 +43,15 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = people.filter(woman => woman.sex === 'f');
+  const isMother = withChildren !== undefined
```

Comment:
It's boolean, also, if you don't have withChildren it means that Boolean(undefined) === false
```suggestion
  const isMother = withChildren
```